run-tests-with-docker-parallel.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+set -u
+
+docker-compose -f docker-compose-parallel.yml build verify-tests
+docker-compose -f docker-compose-parallel.yml up -d --scale firefoxnode=3 selenium-hub firefoxnode 
+echo "Waiting 5s for the nodes to join the selenium hub"
+sleep 5
+docker-compose -f docker-compose-parallel.yml \
+               run \
+               --name verify-tests \
+               -e TEST_ENV=${TEST_ENV:-"joint"} \
+               verify-tests -n 3 -o "--strict -f pretty -f junit -o testreport/"$@
```

Comment:
The `$@` is to pass in extra cucumber args for staging (it omits eIDAS tests since they aren't enabled on staging)
I think it will have to go inside the double quotes so that it gets passed to the `-o` flag

Reply:
ah! good spot, thanks!

Reply:
Also I think you want to use $* - it's a subtle difference and you'll just have to hope none of the params contains a space

Reply:
Actually staging doesn't do this anymore so YAGNI

Reply:
I'll leave (fix) it in just to keep the flexibility for the extra arguments...

Reply:
@rhowe-gds not sure I understand re spaces in `$*` - we used to pass  `--fail-fast --tags "not @Eidas"`